lib/src/user.dart

```diff
@@ -29,14 +29,14 @@ import './app.dart';
 /// locally on the device, and should be treated as sensitive data.
 /// {@category Application}
 class User {
-  final App? _app;
+  App? _app;
```

Comment:
I am not sure why we are changing this. Should we keep the `final` here

Reply:
No. I think it was a mistake. Making this final prevents us from using the `??=` operator in the actual getter.